info.mumble.Mumble.appdata.xml

```diff
@@ -27,7 +27,7 @@
     </screenshot>
   </screenshots>
   <releases>
-    <release type="development" date="2019-03-17" version="1.3.0-rc1"/>
+    <release type="development" date="2019-07-03" version="1.3.0-rc2"/>
```

Comment:
Please append a new release tag here instead of replacing the old one. This part of the file is comparable to a changelog.

Reply:
Oh, that makes sense. Will do.